release.yml

```diff
@@ -40,11 +40,13 @@ stages:
       displayName: 'component detection'
```

Comment:
We can add the condition in this task as well. This task is one of the steps to generate the notice file.